src/views/RuleEngine/components/ResourceDialog.vue

```diff
@@ -114,13 +116,14 @@
 <script>
 import EmqSelect from '~/components/EmqSelect'
 import ArrayEditor from '~/components/ArrayEditor'
+import FileEditor from "~/components/FileEditor"
```

Comment:
It is best to use single quotes uniformly

Reply:
I agree with that.

Reply:
ty I was not aware of. fixed